packages/x-charts/src/BarChart/barClasses.ts

```diff
@@ -0,0 +1,34 @@
+import generateUtilityClass from '@mui/utils/generateUtilityClass';
+import composeClasses from '@mui/utils/composeClasses';
+import generateUtilityClasses from '@mui/utils/generateUtilityClasses';
+
+export interface BarClasses {
+  /** Styles applied to the bar plot element. */
+  root: string;
+  /** Styles applied to the group surrounding a series' bar elements. */
+  series: string;
+  /** Styles applied to the group surrounding a series' labels. */
+  seriesLabels: string;
+}
+
+export type BarClassKey = keyof BarClasses;
+
+export function getBarUtilityClass(slot: string) {
+  return generateUtilityClass('MuiBar', slot);
+}
+
+export const barClasses: BarClasses = generateUtilityClasses('MuiBar', [
+  'root',
+  'series',
+  'seriesLabels',
```

Comment:
Just to share some thaught.

I was considering to propose the following. But it does not work well with classes generation

- `barClasses.root`: removed
- `barClasses.series`: `barLabelClasses.seriesGroup`
- `barClasses.seriesLabels`: `barLabelClasses.seriesGroup`

I've open an issue for v9 on this topic https://github.com/mui/mui-x/issues/18496